x-pack/test/functional/apps/ml/anomaly_detection_jobs/convert_jobs_to_advanced_job.ts

```diff
@@ -392,7 +392,7 @@ export default function ({ getService }: FtrProviderContext) {
         await ml.jobWizardCommon.advanceToSummarySection();
       });
 
-      assertConversionToAdvancedJobWizardRetainsSettingsAndRuns({
+      await assertConversionToAdvancedJobWizardRetainsSettingsAndRuns({
```

Comment:
all three usages of this function are in the scopes of `describe` which aren't async, thus these keywords are illegal here. I wonder though, why is there an assertion after all the "it" sections. It runs before any of the `it`s - it's just very weirdly placed. Was it supposed to be it's own `it` section, or part of the last one, or just an assertion before tests? cc: @qn895

Reply:
```suggestion
     assertConversionToAdvancedJobWizardRetainsSettingsAndRuns({
```

Reply:
I rewrote the test to match mocha recommended design